convert_descriptor_to_swagger/base_tasks.py

```diff
@@ -0,0 +1,126 @@
+def add_metadata(title="Swagger Data Resource") -> dict:
```

Comment:
All of these builder functions should take swag and provide an empty object if none if given.